subprojects/docs/src/docs/userguide/gradleWrapper.adoc

```diff
@@ -170,9 +170,9 @@ Use the Gradle `wrapper` task to generate the wrapper, specifying a version. The
 ++++
 
 [[customizing_wrapper]]
-=== Customizing the Gradle Wrapper
+=== Customizing the Gradle Wrapper Task
 
-Most users of Gradle are happy with the default runtime behavior of the Wrapper. However, organizational policies, security constraints or personal preferences might require you to dive deeper into customizing the Wrapper. Thankfully, the built-in `wrapper` task exposes numerous options to bend the runtime behavior to your needs. Most configuration options are exposed by the underlying task type api:org.gradle.api.tasks.wrapper.Wrapper[].
+Most users of Gradle are happy with the default runtime behavior of the Wrapper. However, organizational policies, security constraints or personal preferences might require you to dive deeper into customizing the Wrapper Task. The built-in `wrapper` task exposes numerous options to bend the runtime behavior to your needs. Most configuration options are exposed by the underlying task type api:org.gradle.api.tasks.wrapper.Wrapper[].
```

Comment:
Let's be even more concrete here. Can you change it to "...into customizing the Wrapper task type."?

Reply:
I'm not convinced this is solving the problem. Shouldn't this be something like: "However, organizational policies, security constraints or personal preferences might require you to customize the `wrapper` task, _so that ..._", with the emphasis on fleshing out the "so that ...". It's not clear what sorts of policies and constraints we're talking about, or what impact they have on the wrapper. Or why folks might want to customise it.